pkgs/applications/science/electronics/nxpmicro-mfgtools/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenv,
+  fetchFromGitHub,
+  cmake,
+  bzip2,
+  libusb1,
+  libzip,
+  openssl,
+  pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "nxpmicro-mfgtools";
+  version = "1.3.191";
+
+  meta = with stdenv.lib; {
+    description = "Freescale /NXP I.MX Chip image deploy tools";
+    longDescription = ''
+      UUU (Universal Update Utility) is a command line tool, evolved out of
+      MFGTools (aka MFGTools v3).
+
+      One of the main purposes is to upload images to I.MX SoC's using at least
+      their boot ROM.
+
+      With time, the need for an update utility portable to Linux and Windows
+      increased. UUU has the same usage on both Windows and Linux. It means the same
+      script works on both OS.
+    '';
+    homepage = https://github.com/NXPmicro/mfgtools;
```

Unquoted URLs are deprecated:

```suggestion
    homepage = "https://github.com/NXPmicro/mfgtools";
```